packages/react-components/react-radio/library/package.json

```diff
@@ -25,6 +25,7 @@
     "@fluentui/react-tabster": "^9.24.6",
     "@fluentui/react-theme": "^9.1.24",
     "@fluentui/react-utilities": "^9.19.0",
+    "@fluentui/semantic-tokens": "0.0.0-alpha.1",
```

Comment:
Have we merged other changes like this? I'm a bit concerned about taking a dependency on an alpha release in a production-ready components.

Reply:
We'll likely be moving these style changes and semantic tokens dep into a custom style hook that is opt-in:
https://github.com/microsoft/fluentui/pull/34735